core/field_image.js

```diff
@@ -116,6 +116,10 @@ Blockly.FieldImage.prototype.maybeAddClickHandler_ = function() {
     this.mouseDownWrapper_ =
         Blockly.bindEventWithChecks_(this.fieldGroup_, 'mousedown', this,
         this.onMouseDown_);
+    // Change the cursor to a pointer if there is a click handler on the image.
+    if (this.imageElement_) {
+      this.imageElement_.style.cursor = 'pointer';
+    }
```

Comment:
is there something in the stylesheet for image elements we could use to do this instead?

Haven't looked closely at when/how often maybeAddClickHandler_ is called, but changing styles in this way can often cause a reflow.